client/src/ui/organisms/article-actions/bookmark-menu/menu-v2.tsx

```diff
@@ -34,6 +36,16 @@ export default function BookmarkV2Menu({ doc }: { doc: Doc }) {
   const [showNewCollection, setShowNewCollection] = useState(false);
   const [disableAutoClose, setDisableAutoClose] = useState(false);
   const [formItem, setFormItem] = useState<Item | NewItem>(defaultItem);
+  const [lastAction, setLastAction] = useState<string>();
```

Comment:
Doesn't this need a default value like `""`?

Reply:
There's no problem with the possible `undefined` value, since we're only ever `===` it - but you're right that supplying a default of `""` means I can avoid the explicit type definition, which is nice.